hlint.cabal

```diff
@@ -83,6 +87,14 @@ library
     else
         cpp-options: -DGPL_SCARES_ME
 
+    if flag(hsyaml) && flag(gpl)
```

Comment:
If you set hsyaml but not gpl, then i think it's reasonable that it uses hsyaml anyway, so I'd simplify this conditional to just hsyaml.

Reply:
Sure, I will update this.